### PR TITLE
Lesson 9 Exercises

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: src/chapters/exceptions/exercises.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Exercises
```
Comment (CarlyLanglois) [inline src/chapters/exceptions/exercises.rst]: This should be Exercises: <section topic>. check the other exercise headers for the format

File: src/chapters/exceptions/exercises.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+=========
+
+To get started, fork and clone this `repo <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn9exceptions>`_ from Github.
```
Comment (CarlyLanglois) [inline src/chapters/exceptions/exercises.rst:4]: Can you make this repo public and not private please?

File: src/chapters/exceptions/exercises.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
+Exercises
+=========
+
+To get started, fork and clone this `repo <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn9exceptions>`_ from Github. 
+
+Divide By Zero!
+---------------
+
+Complete a function called ``Divide()`` in ``Program.cs``.
```
Comment (CarlyLanglois) [inline src/chapters/exceptions/exercises.rst]: image
the way these two sentences get rendered together makes Program.cs.Divide() look like one entity. Change the second to say "the method..."?

File: src/chapters/exceptions/exercises.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+``Divide()`` takes in two parameters: ``x`` and ``y``.
+
+Your function should return the result of ``x/y``.
+
+However, if ``y`` is zero you should throw an appropriate error message.
```
Comment (CarlyLanglois) [inline src/chapters/exceptions/exercises.rst]: change error message to exception. The exception will include a custom error message that they write.

File: src/chapters/exceptions/exercises.rst [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+Try to use an ``ArgumentOutOfRangeException`` and put your ``try/catch`` block in ``Main()`` to test out your error-handling skills!
+
+Test Student Labs
+-----------------
+
+After mentioning to Professor Jackson that you would like to get some more practice with exceptions, she offered to let you write some grading software!
```
Comment (CarlyLanglois) [inline src/chapters/exceptions/exercises.rst:25]: who the heck is this? can you provide more context here if you're telling a story. is this meant to be a theme through the whole page or specifically this task?

in the same vein, is there a connection between these two methods? if there is none, consider placing these in separate files and wrapper classes. we've spent the last several lessons introducing object oriented concepts and it makes sense to practice what we preach.

File: src/chapters/exceptions/exercises.rst [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+Divide By Zero!
+---------------
+
+Complete a function called ``Divide()`` in ``Program.cs``.
+``Divide()`` takes in two parameters: ``x`` and ``y``.
+
+Your function should return the result of ``x/y``.
+
+However, if ``y`` is zero you should throw an appropriate error message.
+Try to use an ``ArgumentOutOfRangeException`` and put your ``try/catch`` block in ``Main()`` to test out your error-handling skills!
```
Comment (CarlyLanglois) [inline src/chapters/exceptions/exercises.rst]: This goes for both of these methods - I think it's a good idea to say explicitly that if an exception is caught, then they should print out the error message

**Review by CarlyLanglois — APPROVED** [review]
Good. I like these two problems! I do think you can tie them together if you want to extend the story that begins in the second part, or make them more distinct with classes - or a combination of both.
Also, just putting this on the table, there are a lot exclamation points on this page.

File: src/chapters/exceptions/exercises.rst (hunk 1 above)
Comment (gildedgardenia) [reply src/chapters/exceptions/exercises.rst]: Fixed!

File: src/chapters/exceptions/exercises.rst (hunk 2 above)
Comment (gildedgardenia) [reply src/chapters/exceptions/exercises.rst:4]: Fixed!

File: src/chapters/exceptions/exercises.rst (hunk 3 above)
Comment (gildedgardenia) [reply src/chapters/exceptions/exercises.rst]: Done!

File: src/chapters/exceptions/exercises.rst (hunk 4 above)
Comment (gildedgardenia) [reply src/chapters/exceptions/exercises.rst]: Done!

File: src/chapters/exceptions/exercises.rst (hunk 5 above)
Comment (gildedgardenia) [reply src/chapters/exceptions/exercises.rst:25]: Changed intro and now the two are better connected!

File: src/chapters/exceptions/exercises.rst (hunk 6 above)
Comment (gildedgardenia) [reply src/chapters/exceptions/exercises.rst]: Done!

**Pull request merged**